roles/insert_kernel_module/tasks/main.yml

```diff
@@ -1,39 +1,19 @@
 ---
-# This role loads one kernel_module specified in "kernel_module"
+# This role loads one kernel module specified in "insert_kernel_module_data"
 
-- fail:
+- name: "Fail on incorrect input : {{ insert_kernel_module_data.name }}"
```

Comment:
What happens when insert_kernel_module_data.name is not defined here? This is what is being checked in this task, right? Won't it fail earlier on the task name then?

Reply:
It would not work, it would fail before displaying the correct text. Fixed in bb75c55